cypress/e2e/tests/pages/generic/home.spec.ts

```diff
@@ -11,6 +11,31 @@ const homeClusterList = homePage.list();
 const provClusterList = new ClusterManagerListPagePo('local');
 const longClusterDescription = 'this-is-some-really-really-really-really-really-really-long-description';
 
+// Reset the home page card prefs, go the home page and ensure the page is fully loaded
+function goToHomePageAndSettle() {
+  // Reset the home page cards pref so that everything is shown
+  cy.setUserPreference({ 'home-page-cards': '{}' });
+
+  cy.intercept('GET', '/v1/provisioning.cattle.io.clusters?exclude=metadata.managedFields', {
+    statusCode: 200,
+    body:       {
+      count: 0,
+      data:  [],
+    },
+  }).as('fetchClustersHomePage');
+
+  // Go to the home page
+  HomePagePo.goToAndWaitForGet();
+  homePage.waitForPage();
+
+  // Wait for the page to settle - filter the cluster list ensures table is ready and page is ready
+  cy.wait('@fetchClustersHomePage');
+
+  // Wait for the cluster table to load and filter so there are no rows
+  homeClusterList.resourceTable().sortableTable().filter('random text');
+  homeClusterList.resourceTable().sortableTable().rowElements().should((el) => expect(el).to.contain.text('There are no rows which match your search query.'));
```

Comment:
If the intent of `goToHomePageAndSettle()` is to ensure that the page is fully loaded, why do we want to filter the table so there's no rows?